packages/workflow-core/src/lib/plugins/external-plugin/individuals-sanctions-v2-plugin/individuals-sanctions-v2-plugin.ts

```diff
@@ -175,7 +201,7 @@ export class IndividualsSanctionsV2Plugin extends ApiPlugin {
           return {
             firstName,
             lastName,
-            dateOfBirth,
+            ...(dateOfBirth && { dateOfBirth }),
```

Comment:
I think this is more safe
```suggestion
            ...(dateOfBirth ? { dateOfBirth } : {}),
```

Reply:
It doesnt really make any difference, just ternary vs && operator.